test/jdk/javax/swing/JColorChooser/Test8152419.java

```diff
@@ -24,6 +24,7 @@
  /*
 * @test
 * @bug 8152419
+* @library /test/lib
 * @summary To Verify JColorChooser tab selection
 * @run main/manual Test8152419
  */
```

Comment:
```suggestion
/*
 * @test
 * @bug 8152419
 * @library /test/lib
 * @summary To Verify JColorChooser tab selection
 * @run main/manual Test8152419
 */
```
The formatting of the jtreg tags looks weird.

Usually, the opening `/*` starts in the first column, and the following `*` are aligned to the first one.

Reply:
I saw that and then forget to change. Updated now.